src/shared/translations/en.json

```diff
@@ -678,7 +678,7 @@
 			"try-currencyA-currencyB-or-currencyC-currencyD": "Try {{currencyA}} / {{currencyB}} or {{currencyC}} / {{currencyD}}"
 		},
 		"frozen": {
-			"message": "{{name}} price limit hit. Click here for more info."
+			"message": "{{currencyKey}} price limit hit."
```

Comment:
"Click here for more info" - doesn't really make sense... (not in the trade pairs, and not in the markets).